test/unit/org/apache/cassandra/schema/CreateTableValidationTest.java

```diff
@@ -48,4 +51,60 @@ public void testInvalidBloomFilterFPRatio() throws Throwable
         // sanity check
         createTable("CREATE TABLE %s (a int PRIMARY KEY, b int) WITH bloom_filter_fp_chance = 0.1");
     }
+
+    @Test
+    public void testCreateTableOnSelectedClusterColumn()
```

Comment:
```suggestion
    public void testCreateTableOnSelectedClusteringColumn()
```
to be corrected on commit